lib/modules/platform/gitlab/index.ts

```diff
@@ -666,15 +667,15 @@ async function tryPrAutomerge(
       ];
       const desiredStatus = 'can_be_merged';
       // The default value of 5 attempts results in max. 13.75 seconds timeout if no pipeline created.
-      const retryTimes = parseInteger(
-        process.env.RENOVATE_X_GITLAB_AUTO_MERGEABLE_CHECK_ATTEMPS,
+      const retryTimes = GlobalConfig.get(
+        'gitlabAutoMergeableCheckAttempts',
```

Comment:
pass this as an initPlatform() parameter and save to local config instead of reading from GlobalConfig.get()